README.md

```diff
@@ -90,7 +90,13 @@ def run(plan, args={}):
     prometheus_url = prometheus-package.run(plan, [service_a_metrics_job])
 
     # start grafana where dashboards are located at github.com/example-org/example-package/static-files/dashboards
-    grafana.run(plan, prometheus_url, "github.com/example-org/example-package/static-files/dashboards")
+    grafana.run(
+        plan,
+        prometheus_url,
+        "github.com/example-org/example-package/static-files/dashboards",
+        # optionally load predefined grafana alerts from file
+        grafana_alerts_file=read_file(src="./static-files/alerts.yaml"),
```

Comment:
any reason you chose to pass the entire file as opposed to the locator to the file like above? not against passing the whole file, but would be nice to have a consistent api as the other args

Reply:
I updated this so it should be consistent now, lmk what you think 😄